tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -1481,10 +1489,19 @@ void cipher_entry_points(int alg_arg, int key_type_arg,
         output[i] = 0xa5;
     }
 
+    mbedtls_test_driver_cipher_hooks.cipher_update_hits = 0;
     status = psa_cipher_encrypt(
         key, alg, input->x, input->len,
         output, output_buffer_size, &function_output_length);
-    TEST_EQUAL(mbedtls_test_driver_cipher_hooks.hits, 1);
+#if defined(MBEDTLS_AES_C)
+    TEST_EQUAL(mbedtls_test_driver_cipher_hooks.cipher_encrypt_hits, 1);
+#else
+    /* The call to psa_cipher_encrypt() is intentionally supposed to fail on the
+     * 1st access to the driver since "forced_status" is set. However this
+     * initial access happens in psa_cipher_update() (random number generation
+     * for IV) so psa_cipher_encrypt() never gets called. */
+    TEST_EQUAL(mbedtls_test_driver_cipher_hooks.cipher_update_hits, 1);
```

Comment:
But we're still not testing the case where the `"cipher_encrypt"` entry point returns an error. What about my `forced_status_encrypt` suggestion? That way, the driver would succeed when called by CTR-DRBG, but would then fail in `cipher_encrypt`.

Reply:
Sorry, I misunderstood your previous comment. I'll fix this :)

Reply:
Oh, thanks for the detailed explanation: yes, I didn't get it previously. I agree with your proposal.